releasenotes/notes/0.20/prepare-0.20-79918ed0fc5b496e.yaml

```diff
@@ -0,0 +1,41 @@
+---
+prelude: |
+  The Qiskit 0.20.0 release highlights are:
+
+  * The introduction of multithreaded modules written in Rust to accelerate
+    the performance of certain portions of Qiskit Terra and improve scaling
+    with larger numbers of qubits. However, when building Qiskit from source a
+    `Rust <https://www.rust-lang.org/>`__ compiler is now required.
+
+  * More native support for working with a :class:`~.Target` in the transpiler.
+    Several passes now support working directly with a :class:`~.Target` object
+    which makes the transpiler robust in the types of backends it can target.
+
+  * The introduction of the :mod:`qiskit.primitives` module. These APIs are
+    used to provide a higher level abstraction for working with
+    :class:`~.QuantumCircuit` objects and using backends for computation. For
+    example, the :class:`~qiskit.primitives.BaseEstimator` defines an abstract
+    interface
+    for estimating an expectation value of a QuantumCircuit and an observable.
+    This can then be used to construct higher level algorithms and applications
+    that are built using the estimation of expectation values without having
+    to worry about the implementation of computing the expectation value.
```

Comment:
```suggestion
  * The introduction of the :mod:`qiskit.primitives` module. These APIs are
    provide different abstraction levels for computing outputs of interest from
    :class:`~.QuantumCircuit` and using backends. For
    example, the :class:`~qiskit.primitives.BaseEstimator` defines an abstract
    interface for estimating an expectation value of an observable.
    This can then be used to construct higher level algorithms and applications
    that are built using the estimation of expectation values without having
    to worry about the implementation of computing the expectation value.
    This decoupling allows the implementation to improve in speed and quality
    while adhering to the defined abstract interface.
    Likewise, the :class:`~qiskit.primitives.BaseSampler` computes
    quasi-probability distributions from circuit measurements. Other primitives will
    be introduced in the future.
```